Gemfile.lock

```diff
@@ -141,18 +141,17 @@ GEM
       execjs
     coffee-script-source (1.12.2)
     concurrent-ruby (1.1.5)
-    config (2.0.0)
-      activesupport (>= 4.2)
+    config (2.2.1)
       deep_merge (~> 1.2, >= 1.2.1)
-      dry-schema (~> 1.0)
+      dry-validation (~> 1.0, >= 1.0.0)
     confstruct (0.2.7)
     coveralls (0.8.23)
       json (>= 1.8, < 3)
       simplecov (~> 0.16.1)
       term-ansicolor (~> 1.3)
       thor (>= 0.19.4, < 2.0)
       tins (~> 1.6)
-    crass (1.0.5)
+    crass (1.0.6)
```

Comment:
hadn't noticed this before, but i think this is my new favorite gem name (as i guessed, CSS related: "a pure Ruby CSS parser based on the CSS Syntax Level 3 spec")